.gitattributes

```diff
@@ -33,3 +33,4 @@ Makefile	eol=lf
 *.wixproj	eol=crlf
 *.wxs	eol=crlf
 *.rtf	eol=crlf
+*.txt	eol=crlf
```

Comment:
Shouldn't this be `text`, like `*.cs`?

Reply:
I think we do need the `eol` setting: https://git-scm.com/docs/gitattributes#Documentation/gitattributes.txt-Settostringvaluecrlf

To test this I did:
```
>  rm .\ThirdPartyNotices.txt
>  git checkout HEAD .\ThirdPartyNotices.txt
```

And then it switches to Windows line endings when I look in notepad++:

![image](https://user-images.githubusercontent.com/840039/59525039-ea636100-8e9a-11e9-972e-8af14636e2ba.png)

Using just `text`, these didn't work.

But the docs aren't exactly clear if someone else knows better?

Reply:
Oh all the `.cs` files on my system are LF...

So maybe we have more than just `.txt` wrong?

Reply:
I'm going to try making ours closer to what corefx has: https://github.com/dotnet/corefx/blob/master/.gitattributes

It may be this part we are missing:
```gitattributes
###############################################################################
# Set default behavior to automatically normalize line endings.
###############################################################################
* text=auto

```